src/components/MultiHopTrade/components/VerifyAddresses/VerifyAddresses.tsx

```diff
@@ -144,12 +144,9 @@ export const VerifyAddresses = () => {
   const fetchAddresses = useCallback(async () => {
     if (!wallet || !sellAssetAccountId || !sellAccountMetadata) return
 
-    const deviceId = await wallet.getDeviceID()
```

Comment:
This was unused for `getReceiveAddress()` and related `getAddress()` methods of sorts across the board.